doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -126,6 +126,13 @@ This, combined with the field ordering [already described in section before](#cu
 avoids deadlocks when two Cygnus instances are trying to upsert in the same table
 (more detail on this in issue [#2197](https://github.com/telefonicaid/fiware-cygnus/issues/2197)).
 
+
+### AlterationType from NGSIv2 subscription
+
+Regarding with NGSIv2 subcriptions based in [alterationType](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#subscriptions-based-in-alteration-type) Cygnus can deal with any subscription containing alterationType as entity attribute. AlterationType received in entity is not progresing as NGSIv2 event, just is interpretated by cygnus lastData funtionality in this way:
```

Comment:
This part:

> AlterationType received in entity is not progresing as NGSIv2 event

Should be also mentioned in the historic sink

Reply:
> This part:
> 
> > AlterationType received in entity is not progresing as NGSIv2 event
> 
> Should be also mentioned in the historic sink

In general none sink based on columnAggregator is going to receive alterationType, just SQL sinks are going to interpretate values or alterationType